Content.Server/Stunnable/StunbatonSystem.cs

```diff
@@ -94,8 +94,11 @@ private void OnThrowCollide(EntityUid uid, StunbatonComponent comp, ThrowDoHitEv
             if (!_cellSystem.TryGetBatteryFromSlot(uid, out var battery) || !battery.TryUseCharge(comp.EnergyPerUse))
                 return;
 
-            StunEntity(args.Target, comp);
-            SendPowerPulse(args.Target, args.User, uid);
+            if (_robustRandom.Prob(comp.OnThrowStunChance))
```

Comment:
this would mean it uses energy even if it doesn't stun? that might be desirable, tho

Reply:
Shoot, I actually did not notice that tryusecharge... I'll fix that.

Reply:
There we go, moved it into the probability if statement